internal/service/backupcompliancepolicy/resource_backup_compliance_policy_test.go

```diff
@@ -9,51 +9,51 @@ import (
 
 	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
 	"github.com/hashicorp/terraform-plugin-testing/terraform"
-
 	"github.com/mongodb/terraform-provider-mongodbatlas/internal/common/conversion"
 	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/acc"
 )
 
-const resourceName = "mongodbatlas_backup_compliance_policy.backup_policy_res"
+const (
+	resourceName   = "mongodbatlas_backup_compliance_policy.backup_policy_res"
+	dataSourceName = "data.mongodbatlas_backup_compliance_policy.backup_policy"
+)
 
-func TestAccGenericBackupRSBackupCompliancePolicy_basic(t *testing.T) {
+func TestAccBackupCompliancePolicy_basic(t *testing.T) {
 	var (
 		orgID          = os.Getenv("MONGODB_ATLAS_ORG_ID")
+		projectName    = acc.RandomProjectName() // No ProjectIDExecution to avoid conflicts with backup comliance policy
 		projectOwnerID = os.Getenv("MONGODB_ATLAS_PROJECT_OWNER_ID")
-		projectName    = acc.RandomProjectName()
 	)
 
-	resource.ParallelTest(t, resource.TestCase{
+	resource.Test(t, resource.TestCase{
```

Comment:
just want to check on the reason behind this change, and potentially leave a comment on why it cannot run in parallel.

Reply:
change to ParallelTest as i'll keep them in their own project